sdk/lib/rtl/amd64/slist.S

```diff
@@ -160,29 +160,34 @@ ExpInterlockedPopEntrySListEnd:
     ret
 
 RtlInterlockedPopEntrySListEmpty:
+
     xor rax, rax
     ret
 
-RtlInterlockedPopEntrySList16:
+ENDFUNC
+
+
+FUNC RtlInterlockedPopEntrySList16
+    mov [rsp + P3Home], rbx
```

Comment:
Why P3Home ? This makes it feel like rsp points to a CONTEXT or a TRAP_FRAME.

Reply:
That is just the parameter home space. Every function has it. And every function is free to store in it, whatever it wants to.
See https://docs.microsoft.com/en-us/cpp/build/stack-usage?view=msvc-160

Reply:
Indeed.